src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -90,6 +90,24 @@ package
     private function bufferingChangeHandler(e:BufferEvent):void
     {
     }
+
+    /**
+     * Send messages to the browser console log.In future this can be hooked to any other Debugging tools.
+     * @private
+     * @method AkamaiHDPlayer#SendToDebugger
+     * @param {string} value The value to be passed to the browser console.
+     * @param {string} referrer The fuction or process which passed the value.
+     * @param {string} channelBranch It can be info, debug, warn, error or log.
+     * @returns {boolean} True or false indicating success
+     */
+    private function SendToDebugger(value:String, referrer:String = null, channelBranch:String = "log"):Boolean
+    {
+      var channel:String = "OO." + channelBranch;
```

Comment:
this would be really good if OO.error and OO.warn existed but right now we only have OO.log.  You can log errors and warn with console.* but standard logs should be logged with OO.log